examples/sensor-data-scala/README.md

```diff
@@ -49,23 +49,34 @@ wrk -c 400 -t 400 -d 500 -s wrk-04-moderate-breeze.lua <http-ingress-url>
 ```
 
 
-### Example Deployment on GKE
+### Example Deployment on Kubernetes
 
-* Make sure you have installed a GKE cluster with Cloudflow running as per the [installation guide](https://github.com/lightbend/cloudflow-installer).
-Make sure you have access to your cluster:
+* Make sure you have installed a Kubernetes cluster with Cloudflow running as per the [installation guide](https://github.com/lightbend/cloudflow-installer).
+
+If you are using a GKE cluster, make sure you have access to your cluster:
```

Comment:
Would this not also hold for any other type of cluster?

Reply:
Yeah let me rephrase that

Reply:
Fixed